x-pack/plugin/mapper-constant-keyword/build.gradle

```diff
@@ -1,7 +1,8 @@
 import org.elasticsearch.gradle.internal.info.BuildParams
 
 apply plugin: 'elasticsearch.internal-es-plugin'
-apply plugin: 'elasticsearch.legacy-yaml-rest-test'
+apply plugin: 'elasticsearch.internal-yaml-rest-test'
+apply plugin: 'elasticsearch.internal-test-artifact'
```

Comment:
The test artifact plugin shouldn't be required here. This is only necessarily if another project needs to consume the tests of this project as an artifact dependency.